src/components/CustomSidebar.jsx

```diff
@@ -0,0 +1,123 @@
+import { Avatar, Box, Typography } from "@mui/material";
+import {
+  ProSidebar,
+  Menu,
+  MenuItem,
+  SubMenu,
+  SidebarContent,
+} from "react-pro-sidebar";
+import Divider from '@mui/material/Divider';
+import Profile from "../assets/images/profile.png";
+import Dashboard from "../assets/images/Dashboard.png";
+import Leads from "../assets/images/leads.png";
+import Buyers from "../assets/images/buyers.png";
+import Clients from "../assets/images/client.png";
+import Properties from "../assets/images/properties.png";
+import Orders from "../assets/images/orders.png";
+import RedBox from "../assets/images/redbox.png";
+import CallBack from "../assets/images/callback.png";
+import Investments from "../assets/images/investments.png";
+import Payment from "../assets/images/payments.png";
+import Magik from "../assets/images/magik.png"; 
+
+
+function SideNav() {
+  return (
+    <div className="container" style={{
```

Comment:
Don't use `div`, you are using mui, so use `Box`

Reply:
Yes deepak, its an old habit..i forget to change to box sometimes..will see to that this won't happen again